packages/@aws-cdk/aws-s3-assets/lib/asset.ts

```diff
@@ -5,11 +5,62 @@ import * as cdk from '@aws-cdk/core';
 import * as cxapi from '@aws-cdk/cx-api';
 import * as fs from 'fs';
 import * as path from 'path';
+import { BundlingDockerImage, DockerVolume } from './bundling';
 
 const ARCHIVE_EXTENSIONS = [ '.zip', '.jar' ];
+const BUNDLING_INPUT_DIR = '/input';
+const BUNDLING_OUTPUT_DIR = '/output';
```

Comment:
maybe `/cdk-input` and `/cdk-output` to avoid as much as possible a collision with existing directories coming from Docker images?

Reply:
Let's do `/asset-input` and `/asset-output`.